### ISSUE TITLE
Heading elements are not in a sequentially-descending order

### ISSUE THREAD

**Issue by andrew-t-james** [Vets-Who-Code/vets-who-code-app#215]
**Describe the bug**
Heading elements are not in a sequentially-descending order on the homepage. See [here](https://web.dev/heading-order/?utm_source=lighthouse&utm_medium=devtools)

**To Reproduce**
1. Go to vetswhocode.io
2. Run lighthouse report
3. View accessibility report.

**Expected behavior**
The element should be ordered correctly based on the current markup. 

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**
